components/shared/tools/ToolHelperComponents.tsx

```diff
@@ -0,0 +1,159 @@
+import { FeaturedTool } from '@/model/tools';
+import { AiOutlineSlack } from 'react-icons/ai';
+import { BsEye } from 'react-icons/bs';
+import { FaGithub, FaGitlab, FaHubspot, FaPaintBrush } from 'react-icons/fa';
+import {
+  GoBrowser,
+  GoFileDirectory,
+  GoGlobe,
+  GoQuestion,
+  GoSearch,
+} from 'react-icons/go';
+import {
+  PiMicrosoftExcelLogo,
+  PiMicrosoftOutlookLogoDuotone,
+} from 'react-icons/pi';
+import { SiJson, SiNotion } from 'react-icons/si';
+
+// note(tylerslaton) - This will eventually be retrieved from the tools site, for now we must endure the pain of hardcoding.
+const featuredTools: FeaturedTool[] = [
```

Comment:
This list is being maintained in two different places. Can this be DRY-ed up?

Reply:
Meant to delete that file, thanks